src/wizard/javascript/sveltekit/index.md

```diff
@@ -0,0 +1,50 @@
+---
+name: SvelteKit
+doc_link: https://docs.sentry.io/platforms/javascript/guides/sveltekit/
+support_level: production
+type: framework
+---
+
+Configure your app automatically with the [Sentry wizard](https://docs.sentry.io/platforms/javascript/guides/sveltekit/#install).
+
+```bash
+npx @sentry/wizard@latest -i sveltekit
+```
+
+Sentry wizard will automatically patch your application:
+
+- Create or update `src/hooks.client.js` and `src/hooks.server.js` with the default `Sentry.init` call and Sentry's hooks handlers.
+- Update `vite.config.js` to add source maps upload and auto-instrumentation via Vite plugins.
+- create `.sentryclirc` and `sentry.properties` files with configuration for sentry-cli (which is used when automatically uploading source maps).
+
+You can also [configure the Sentry SDK manually](https://docs.sentry.io/platforms/javascript/guides/sveltekit/manual-setup/).
+
+Configure the Sentry SDK:
+
+To configure the Sentry SDK, edit the `Sentry.init` options in `hooks.(client|server).(js|ts)`:
+
+```javascript
+import * as Sentry from '@sentry/sveltekit';
+
+Sentry.init({
+  dsn: '___PUBLIC_DSN___',
+  integrations: [new Sentry.BrowserTracing()],
+  // Set tracesSampleRate to 1.0 to capture 100%
+  // of transactions for performance monitoring.
+  // We recommend adjusting this value in production
+  tracesSampleRate: 1.0,
+});
+```
+
+The above configuration captures both error and performance data. We recommend adjusting `tracesSampleRate` in production, see [Sampling](https://docs.sentry.io/platforms/javascript/configuration/sampling/).
+
```

Comment:
I think we don't want to do this here because generally what we're showing in this onboarding step is the easiest way to set up the SDK. Which is the one-line wizard command. If users run the wizard, `handleErrorWithSentry` should already be added to the hooks files. (We nevertheless link to the manual setup if they want to avoid the wizard).